ranger/container/settings.py

```diff
@@ -100,6 +100,8 @@
     'w3m_offset': int,
     'wrap_scroll': bool,
     'xterm_alt_key': bool,
+    'show_titlebar': str,
```

Comment:
These should be `bool` values, not `str`.

Reply:
I did that first. But then I looked at other options which also used 'false' / 'true' as `str`. And when I used a `boolean` you always had to write True / False with a capital letter in the beginning which I think from a user perspective is not too practical.

But I'll change it :)